release/models/interfaces/openconfig-interfaces.yang

```diff
@@ -931,6 +937,304 @@ revision "2023-02-06" {
     }
   }
 
+  grouping subinterface-counters-state {
```

Comment:
Yes, it seems to be the case -- please split the groupings so that there are not duplicate leaf definitions.

Generally, you can have:

```yang

grouping interface-counters {
   uses interface-counters-common;
   uses interface-counters;
}

grouping subinterface-counters {
   uses interface-counters-common;
   uses subinterface-counters;
}
```